src/events/eventToAction.js

```diff
@@ -145,6 +146,12 @@ export default (state: GlobalState, event: Object): EventAction => {
         type: opToActionUserGroup[event.op],
       };
 
+    case 'user_status':
+      return {
+        ...event,
+        type: EVENT_USER_STATUS_UPDATE,
+      };
```

Comment:
Take a look at the block of 8 event types up above, with `actionTypeOfEventType`. This should go there.

Reply:
Oh, didn't notice these `case` refactorings.
I am not sure this is an improvement, removing simple duplication by increasing the complexity of the code...
Anyway, did the change.

Reply:
Cool.

> I am not sure this is an improvement, removing simple duplication by increasing the complexity of the code...

I think the main thing I liked about this particular small refactoring was that we had all these different cases which were doing essentially the same thing... but that wasn't obvious because they were scattered around this `switch` statement and interleaved with cases that worked differently. And so each of them added complexity as you had to look at each case individually to see what it did. Now it's very clear that these all have the exact same boring structure, and the exceptions which do something more interesting stand out better.

There are other ways that could be done. Simply reordering cases to bring these together would have accomplished that and certainly not added any complexity, and would have been fine though remained verbose. When I went to make it less verbose, I chose this particular style largely because... this is what the existing code in the file already does for a bunch of other cases! :slightly_smiling_face: